dev/cnf/oss_dependencies.maven

```diff
@@ -120,8 +120,8 @@ io.smallrye.common:smallrye-common-classloader:1.5.0
 io.smallrye.common:smallrye-common-constraint:1.5.0
 io.smallrye.common:smallrye-common-expression:1.5.0
 io.smallrye.common:smallrye-common-function:1.5.0
-io.smallrye.config:smallrye-config-common:2.0.0-RC1
-io.smallrye.config:smallrye-config:2.0.0-RC1
+io.smallrye.config:smallrye-config-common:2.0.0-RC4
```

Comment:
The line is no longer correct. It should be
io.smallrye.config:smallrye-config-core:2.0.0-RC4

Reply:
please remove the lines 
io.smallrye.common:smallrye-common-annotation:1.5.0
io.smallrye.common:smallrye-common-classloader:1.5.0
io.smallrye.common:smallrye-common-constraint:1.5.0
io.smallrye.common:smallrye-common-expression:1.5.0
io.smallrye.common:smallrye-common-function:1.5.0

Reply:
after I talked with @Joseph-Cass , we do need this as they are the dependencies needed for resolving smallrye-config-core